src/components/modern/LargeTokenValueView.js

```diff
@@ -25,54 +25,71 @@ import { BigNumber } from 'bignumber.js';
 
 // Components
 import { Spacing } from 'components/modern/Layout';
-import Text from 'components/modern/Text';
+import TokenValueView from 'components/modern/TokenValueView';
+import FiatValueView from 'components/modern/FiatValueView';
 
 // Utils
-import { formatTokenValueWithoutSymbol } from 'utils/format';
 import { appFont, spacing } from 'utils/variables';
+import { useThemeColors } from 'utils/themes';
+import { getAssetValueInFiat } from 'utils/rates';
+import { wrapBigNumber } from 'utils/bigNumber';
+
+// Selectors
+import { useFiatCurrency, useChainRates } from 'selectors';
 
 // Types
 import type { TextStyleProp } from 'utils/types/react-native';
-
+import type { Chain } from 'models/Chain';
 
 type Props = {|
   value: ?BigNumber,
+  assetAddress: string,
+  chain: Chain,
   symbol: string,
   style?: TextStyleProp,
 |};
 
 /**
  * Large (& stylized) component to display token value.
  */
-function LargeTokenValueView({ value, symbol, style }: Props) {
-  if (!value) return null;
+function LargeTokenValueView({ value, assetAddress, chain, symbol, style }: Props) {
```

Comment:
I think that this components should be renamed to reflect its new design: `LargeFiatTokenValueView`?

Reply:
Done